config_defaults_inc.php

```diff
@@ -851,13 +851,27 @@
 $g_window_title = 'MantisBT';
 
 /**
+ * OpenSearch engine title prefix.
+ * This is used to describe Browser Search entries, and must be short enough
+ * so that when inserted into the 'opensearch_XXX_short' language string, the
+ * resulting text is 16 characters or less, to be compliant with the limit for
+ * the ShortName element as defined in the OpenSearch specification.
+ * @link http://www.opensearch.org/Specifications/OpenSearch/1.1
+ * @see $g_window_title
+ * @global string $g_search_title
+ */
+$g_search_title = '%window_title%';
```

Comment:
> evaluate how much we'd gain with your proposed approach.

Quick and dirty removal of config_eval
https://github.com/atrol/mantisbt/tree/config_eval_remove

There is a bit less memory usage (removed the caching of the evaled options) and there is about 7% better performance in 2nd benchmark, see results below.
Not outstanding, but not that bad for such a small part of the code.

Do you think it's worth to think more about it (obsolete cookie seetings, ...) and target it for a version > 2.0?

My View 2nd call
------------------

Original master-1.3.x
Page execution time: 0.3540 seconds
Memory usage: 10,411 KiB

Removed config_eval
Page execution time: 0.3499 seconds
Memory usage: 10,388 KiB

View Issues 2nd call
--------------------

Original master-1.3.x
Page execution time: 0.2916 seconds
Memory usage: 10,514 KiB

Removed config_eval
Page execution time: 0.2705 seconds
Memory usage: 10,441 KiB